infrastructure/tooling/src/generate/generators/db-schema.js

```diff
@@ -10,6 +10,6 @@ exports.tasks = [{
   run: async (requirements, utils) => {
     const schema = Schema.fromDbDirectory(path.join(REPO_ROOT, 'db'));
 
-    writeRepoJSON('generated/db-schema.json', JSON.parse(schema.asSerializable()));
+    writeRepoJSON('generated/db-schema.json', schema.asSerializable());
```

Comment:
How do we guarantee that the schema won't change between `yarn generate` if we remove `stringify(...)` from the `asSerializable` method?

Reply:
Nevermind, I just saw that the following commit uses `Map.prototype.values` to keep track of insertion order. Smart!

Reply:
Yep, and also, `writeRepoJson` uses `json-stable-stringify`, so it should still be stable.

Reply:
Will the data still be readable? Previously, we used `{ space: 2 }`.

Reply:
Ah, ok!